ColabTool/Shared/Post/index.jsx

```diff
@@ -3,8 +3,8 @@ import {
   Card,
 } from '@bufferapp/components';
 import style from './style.css';
-import PostButtonPanel from '../PostButtonPanel';
 import PostDetails from '../PostDetails';
+import PostTopDetails from '../PostTopDetails';
```

Comment:
Now that we have a clearer example of these things, `PostTopDetails` might be better named `PostHeader` and `PostDetails` might be better named `PostFooter`.